src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsstafflookupservice/client/MicrosoftGraphClient.kt

```diff
@@ -1,21 +1,23 @@
 package uk.gov.justice.digital.hmpps.hmppsstafflookupservice.client
 
 import com.fasterxml.jackson.annotation.JsonProperty
+import kotlinx.coroutines.reactor.awaitSingle
 import org.springframework.beans.factory.annotation.Qualifier
 import org.springframework.stereotype.Service
 import org.springframework.web.reactive.function.client.WebClient
-import org.springframework.web.reactive.function.client.awaitBody
+import reactor.netty.http.client.HttpClientRequest
+import java.time.Duration
 
 data class UserResponse(
   @JsonProperty("@odata.nextLink") val nextLink: String? = null,
   val value: List<MicrosoftADUser>,
 )
 
 data class MicrosoftADUser(
-  val givenName: String,
-  val surname: String,
-  val jobTitle: String? = null,
-  val mail: String? = null,
+  val givenName: String?,
```

Comment:
I found from postman that these are actually all optional except for the principal